swagger.yml

```
@@ -3063,6 +3067,27 @@ paths:
       tags:
         - "Partner API v3"
       summary: Get specific content
+      description: |
+        ### Content metadata
+
+        Get metadata about a specific content in JSON format.
+
+        If the interpreter service is enabled for a recipient, it can add structured metadata to content which has been received.
+        The `interpreted` label indicates if the interpreter has run for a given content.
+        Interpreted data is never used in place of metadata provided by the sender or Kivra, it can only be added.
+
+        Any property of the response that was added by the interpreter is clearly identified as such.
+        If an interpreted property is an object, it will have an `interpreted` property which is `true`.
+        If an interpreted property is a primitive value, its key path will be present in the `interpreted_fields` property.
+        See the response schema below for details.
+
+        ### Content file data
+
+        A given content can contain multiple files ("parts"), to download one, pass the `part` query parameter.
```

```suggestion
        A given content can contain multiple files ("parts"). Fetching of such parts is done by passing the `part` query parameter.
```
Not sure "download" is the word to use.